x-pack/plugins/observability/public/components/shared/exploratory_view/configurations/constants/constants.ts

```diff
@@ -105,4 +107,9 @@ export const ReportToDataTypeMap: Record<ReportViewTypeId, AppDataType> = {
   mem: 'infra_metrics',
   logs: 'infra_logs',
   cpu: 'infra_metrics',
+  cwv: 'ux',
```

Comment:
Could you include this next to the other ux bits, just for easy referencing in the future.